### PR TITLE
Home page and Category page UI 

### PR DESCRIPTION
**Pull request by nikhiljugale007**
Added home page UI and category page UI .
Files to check:
Home.jsx
Header.jsx
Footer.jsx
App.jsx
QuizCard.jsx
Category.jsx

### CODE REVIEW

File: src/components/QuizCard/QuizCard.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	transform: scale(1.1);
+}
+.category-card img {
+	height: 25rem;	
+      border-radius: 10px;
+}
+.category-card-text {
+	position: absolute;
+	bottom: 16px;
+	left: 16px;
```
Comment (omkarpatke) [inline src/components/QuizCard/QuizCard.css]: suggestion: instead of px use rem here.😊

File: src/pages/home-page/Home.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	background-color: #6077ef;
+	color: #fff;
+	padding: 2rem;
+	gap: 1rem;
+	border-radius: 8px;
+	width: 25rem;
+}
+.hero-card .btn-primary {
+	color: #fff;
+	background-color: #7589f0;
```
Comment (omkarpatke) [inline src/pages/home-page/Home.css]: suggestion: use variables for color to maintain consistency.

File: src/components/Header/Header.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.headerr {
```
Comment (omkarpatke) [inline src/components/Header/Header.css:1]: suggestion: use a better class name here.

**Review by omkarpatke — APPROVED** [review]
The folder Structure is Good.
Suggestion: use better class names.
after suggested minor changes, you can merge the PR.

**Review by Naman-Saxena1 — APPROVED** [review]
Code looks great, I really like color combination and card that you are using for thriller, looks cool 👍

**Comment by varun-Tad** [comment]
Screenshot 2022-05-16 at 11 25 55 PM


The code looks good to merge. Coming to the UI.Not sure if it's intentional but you can probably add some padding in the block containing the sentence "100 + questions to practice".

File: src/components/QuizCard/QuizCard.css (hunk 1 above)
Comment (nikhiljugale007) [reply src/components/QuizCard/QuizCard.css]: changed px to rems

File: src/pages/home-page/Home.css (hunk 2 above)
Comment (nikhiljugale007) [reply src/pages/home-page/Home.css]: done

**Comment by nikhiljugale007** [comment]
> Screenshot 2022-05-16 at 11 25 55 PM
> 
> The code looks good to merge. Coming to the UI.Not sure if it's intentional but you can probably add some padding in the block containing the sentence "100 + questions to practice".

added some padding around text.

**Pull request merged**